lambdas/ci-mapping/src/ci-mapping-handler.ts

```diff
@@ -44,10 +48,23 @@ const getCIsForHmrcErrors = (event: CiMappingEvent): Array<string> => {
           .map((value) => value.trim())
           .includes(hmrcError)
       )
-      .map(() => ciValue.trim());
+      .map((hmrcError) => ({ ci: ciValue.trim(), reason: hmrcError.trim() }));
```

Comment:
I'm a little confused about casing here, is there a reason for when you'd use snake_case or camelCase?

Reply:
```
									"ci_mapping.$": "States.StringSplit($.parameters.contraindicationMappings,'||')",
									"hmrc_errors.$": "$.check-attempts-exist.Items[*].text.S",
                  "ci_reason_mapping.$": "States.StringToJson($.parameters.contraindicatorReasonMappings)"
```
for the inputs from the statemachine, I suppose would change these `camelCase` and then it would be uniform. But maybe a follow up PR